reactjs/components/atoms/Paragraph/List/List.js

```diff
@@ -37,8 +38,8 @@ class List extends React.Component {
             <Wrapper>
               {list.map((item, index) => (
                 type === 'list_checkbox' ?
-                  <Checkbox key={index} label={item}/> :
-                  <li key={index}><span>{item}</span></li>
+                  <Checkbox key={md5(item)} label={item}/> :
```

Comment:
@andreityuhai great start! 

I'd suggest to enhance it a bit to avoid md5() completely and remove it from imports. Note that you already have `id` property for `<Checkbox />` which is not used yet. You could make it unique by using concatenation of current paragraph id and item index. That unique value should work for both key and id props.